src/AbstractSQLRules2SQL.ts

```diff
@@ -1000,6 +1002,10 @@ const typeRules: Dictionary<MatchFn> = {
 		return `COALESCE(JSON_AGG(${field}), '[]')`;
 	},
 	Equals: Comparison('Equals'),
+	EqualsAny: (args, indent) => {
+		checkArgs('EqualsAny', args, 2);
+		return `${AnyValue(getAbstractSqlQuery(args, 0), indent)} = ANY(${AnyValue(getAbstractSqlQuery(args, 1), indent)})`;
```

Comment:
Ideally it'd be possible to do 
```suggestion
		return `${AnyValue(getAbstractSqlQuery(args, 0), indent)} = ${typeRules.Any(getAbstractSqlQuery(args, 1), indent)}`;
```
to reuse the same `Any` node for the relevant part, although I'm fine with it as-is and leave that as a potential future thing

Reply:
@Page- They are currently fairly similar, the only reason why can't currently be done this way is because `Any` requires a innerType to be casted, which I am not sure we want to infer for `in`. Ofc I can rework the Any logic to be able to handle no parameter on the innerTyping but at this point we could just use the `AnyNode` anyway. I think if we decide to do it, it should be part of a follow up PR

Reply:
Yeah, I just wanted to raise awareness to consider that for future